features/datachoices/src/main/java/org/opennms/features/datachoices/internal/UsageStatisticsReporter.java

```diff
@@ -264,6 +268,10 @@ public UsageStatisticsReportDTO generateReport() {
         usageStatisticsReport.setRequisitionCount(getDeployedRequisitionCount());
         usageStatisticsReport.setRequisitionWithChangedFSCount(getDeployedRequisitionWithModifiedFSCount());
         usageStatisticsReport.setBusinessEdgeCount(m_businessServiceEdgeDao.countAll());
+        usageStatisticsReport.setSinkStrategy(SinkStrategy.getSinkStrategy().getDescr());
```

Comment:
Do we need descr or name here ?

Reply:
I went with description because it was more verbose, but I'm fine with using either.

Reply:
name seems to be better here.